<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EGMServer</class>
 <widget class="QDialog" name="EGMServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot record</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>机器人设置</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>35</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>机器人类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>75</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>传输协议</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboRobotType">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>35</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>ABB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KUKA</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboTransmitProtocol">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>75</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tcp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Udp</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="RobotReceiveText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>171</y>
     <width>541</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>401</y>
     <width>541</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnSaveData">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>保存数据</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearData">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>清除数据</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RobotPort"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnStratOrStopListen">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>开始监听</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboRobotType</tabstop>
  <tabstop>comboTransmitProtocol</tabstop>
  <tabstop>RobotReceiveText</tabstop>
  <tabstop>RobotPort</tabstop>
  <tabstop>btnStratOrStopListen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
